schedule/migration/online_spvm_Upgrade.yaml

```diff
@@ -0,0 +1,81 @@
+name:   online_spvm_Upgrade
```

Comment:
I would prefer to have all in lower letter, I have seen many issues with this in the past:
s/Upgrade/upgrade/ but if breaks current nomenclature, leave it like that.